config_src/infra/FMS2/MOM_io_infra.F90

```diff
@@ -1572,11 +1572,13 @@ subroutine categorize_axes(fileObj, filename, ndims, dim_names, is_x, is_y, is_t
       if (variable_exists(fileobj, trim(dim_names(i)))) then
         if (variable_att_exists(fileobj, trim(dim_names(i)), "cartesian_axis")) then
           call get_variable_attribute(fileobj, trim(dim_names(i)), "cartesian_axis", cartesian)
-          cartesian = adjustl(cartesian)
-          if ((index(cartesian, "X") == 1) .or. (index(cartesian, "x") == 1)) is_x(i) = .true.
-          if ((index(cartesian, "Y") == 1) .or. (index(cartesian, "y") == 1)) is_y(i) = .true.
-          if ((index(cartesian, "T") == 1) .or. (index(cartesian, "t") == 1)) is_t(i) = .true.
+        elseif (variable_att_exists(fileobj, trim(dim_names(i)), "axis")) then
+          call get_variable_attribute(fileobj, trim(dim_names(i)), "axis", cartesian)
```

Comment:
@marshallward and @uramirez8707 what should the behavior be if neither "axis" nor "cartesian_axis" is set for any restart variable?  Error out? Assume "axis"? Something else?

Reply:
This might be a better question for @Hallberg-NOAA.  I'm unsure what the expectations are when files deviate from CF or GFDl conventions.

Reply:
An examination of this routine in its entirety shows that if the axes can not be categorized based on the presence of an appropriate axis name or other information, and then issues a warning if no appropriate information can be found.  So the right answer for this particular snippet of code is not to set any of the `is_[xyt]` flags, and then let the subsequent code try to figure out which axis corresponds with which.

Reply:
All of my concerns would be addressed by simply adding the line `cartesian = ""` between lines 1572 and 1573 of MOM_io_infra.F90.